src/geometry.rs

```diff
@@ -156,6 +213,7 @@ impl<T> Size<T> {
 }
 
 impl Size<f32> {
+    /// A [`Size`] with zero width and height
     pub fn zero() -> Self {
```

Comment:
I wonder if we can make this a constant in the future

Reply:
Yep, 100%.